storybook/src/stories/Form/FileManagement/AreaUploadManager.stories.tsx

```diff
@@ -27,6 +27,7 @@ export const _AreaUploadManager = () => {
   const fileIcons = object('File Icons', defaultIcons)
   const showValue = action('Input Callback');
   const allowedFileTypes = object('allowedFileTypes', fileTypes);
```

Comment:
We can also change it to 
```suggestion
  const allowedFileTypes = object('Allowed File Types', fileTypes);
```

Reply:
Done